app/src/main/java/com/keylesspalace/tusky/adapter/TabAdapter.kt

```diff
@@ -86,21 +88,43 @@ class TabAdapter(private var data: List<TabData>,
         if (holder.itemView.removeButton != null) {
             holder.itemView.removeButton.isEnabled = removeButtonEnabled
             ThemeUtils.setDrawableTint(
-                holder.itemView.context,
-                holder.itemView.removeButton.drawable,
-                (if (removeButtonEnabled) android.R.attr.textColorTertiary else R.attr.textColorDisabled)
+                    holder.itemView.context,
+                    holder.itemView.removeButton.drawable,
+                    (if (removeButtonEnabled) android.R.attr.textColorTertiary else R.attr.textColorDisabled)
             )
         }
 
         if (!small) {
 
             if (data[position].id == HASHTAG) {
                 holder.itemView.chipGroup.show()
-                holder.itemView.actionChip.text = data[position].arguments[0]
 
-                holder.itemView.actionChip.setChipIconResource(R.drawable.ic_edit_chip)
+                data[position].arguments.forEachIndexed { i, arg ->
+
+                    val chip = holder.itemView.chipGroup.getChildAt(i).takeUnless { it.id == R.id.actionChip } as Chip?
```

Comment:
It took me a while to understand what's going on here, why `takeUnless` and why `holder.itemView.chipGroup.size - 1`. A comment would be nice.